trieste/models/gpflow/models.py

```diff
@@ -236,6 +251,16 @@ def evaluate_loss_of_model_parameters() -> tf.Tensor:
 
         multiple_assign(self.model, current_best_parameters)
 
+    def trajectory_sampler(self) -> TrajectorySampler:
+        """
+        Return a trajectory sampler. For :class:`GaussianProcessRegression`, we build
+        trajectories using a random Fourier feature approximation.
+
+        :return: The trajectory sampler.
+        """
+        models_data = Dataset(self.model.data[0].value(), self.model.data[1].value())
+        return RandomFourierFeatureTrajectorySampler(self, models_data, self._num_rff_features)
```

Comment:
this works only with GPR at the moment and that's why its here and not in the `GPflowPredictor interface`?